kv/dist_sender_test.go

```diff
@@ -376,7 +376,9 @@ func TestRetryOnDescriptorLookupError(t *testing.T) {
 	ctx := &DistSenderContext{
 		rpcSend: testFn,
 		rangeDescriptorDB: mockRangeDescriptorDB(func(_ proto.Key, _ lookupOptions) (_ []proto.RangeDescriptor, err error) {
-			err, errors = errors[0], errors[1:]
```

Comment:
fair enough, but this pattern is quite common, i don't think we want to change it everywhere